docs/walkthrough.md

```diff
@@ -138,7 +138,16 @@ chmod +x ./kubectl
 We'll assume hereafter that all `kubectl` commands are using this
 newly-installed executable.
 
-To configure `kubectl` to communicate with the service catalog APU server:
+## Step 5 - Configuring `kubectl` to Talk to the API Server
+
+To configure `kubectl` to communicate with the service catalog API server, we'll have to
+get the IP address of the `Service` (and just the IP - no ports or `http://` prefixes)
```

Comment:
@krancour you're right, but I am not going to mention the `LoadBalancer` case here, because, like @MHBauer said, we're assuming they used the `helm install` command above, which uses the default `NodePort` example